packages/grid/x-data-grid/src/hooks/features/rows/gridVisibleRowsInterfaces.ts

```diff
@@ -0,0 +1,12 @@
+import type { GridRowId } from '../../../models/gridRows';
```

Comment:
Nit: Should this interface and `gridVisibleRowsSelectors` be moved in close proximity to the rest of the code `hooks/features/filter/gridFilterState` or `models/gridVisibleRows` for better organization of files as it's being used in `useGridFilter`?